programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -2252,6 +2280,14 @@ void scm_service_stop() {
 
     remove_all_nrpt_rules();
 
+    // tun close needs to be fixed
```

Comment:
what's wrong with tun_close? can we fix it now in this PR?  If not, we should create an issue in GH and maybe make this comment a TODO that references the issue.

Reply:
When we call tun_close, the ziti_loop thread is crashing. It looks like it is not killing the sessions gracefully. I will file an issue against this repo for this error.

Reply:
https://github.com/openziti/ziti-tunnel-sdk-c/issues/275

Reply:
please add the GitHub issue link as a comment, so it survives this PR.